web/html/src/components/picker/recurring-event-picker.tsx

```diff
@@ -0,0 +1,484 @@
+import * as React from "react";
+
+import { DateTimePicker } from "../datetimepicker";
+import { Combobox } from "../combobox";
+import { ComboboxItem } from "../combobox";
+import { Form } from "components/input/Form";
+import { Text } from "components/input/Text";
+import { Utils } from "utils/functions";
+import styles from "./recurring-event-picker.css";
+
+type RecurringType = "hourly" | "daily" | "weekly" | "monthly" | "cron";
+// TODO: This should be `Record<string, string>` or `Record<string, number>`, but currently they're used mixed up
```

Comment:
I didn't took a deep look into this: how hard/long could it be to address and fix this? Would it be worth of doing it right now or we can live with it?

Reply:
This would require functional changes, something that I'm trying to avoid in the migration since it would require considerably deeper testing of everything related. Essentially the problem is that some places set the values as strings while others set it as numbers and it isn't trivial to refactor this without testing it through the full stack.  

Since this code works right now, I don't see a need to put this time and effort into here right now, but this TODO is a good option for anyone who happens to work more in depth on this piece in the future.